Gemfile

```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 
 gem "rails", ">= 4.0.0"
-gem "kaminari"
+# FIXME: Fix garage to work with kaminari v1.
+gem "kaminari", "< 1"
```

Comment:
Hmm... I found some usages of kaminari's methods in main code (`per_page` and `page`). I'm not sure but we have to require kaminari gem as runtime dependency in this case probably?